tests-e2e/fixture/fixture.go

```diff
@@ -333,6 +334,48 @@ func cleanUpOldDeploymentTargetAPIs(ns string, k8sClient client.Client) error {
 	return nil
 }
 
+func cleanUpOldDeploymentTargetClassAPIs(ns string, k8sClient client.Client) error {
```

Comment:
DeploymentTargetClass is cluster-scoped, so we don't need to restrict the list to a namespace.

Reply:
OK. Removed, thanks